### ISSUE TITLE
Load cpp tutorial

### ISSUE THREAD

**Issue by Doutyn** [wak3upalex/cppedu#1]
Textbooks on C++ are essential tools for anyone looking to learn this versatile programming language. They cover a wide range of topics, from basic syntax and control structures to more advanced concepts like object-oriented programming and template metaprogramming

**Comment by wak3upalex** [comment]
What type of tutorial, you want to see?

**Comment by Doutyn** [comment]
"Effective Modern C++" by Scott Meyers

**Comment by wak3upalex** [comment]
Do you mean to add a list of sources of information?

**Comment by Doutyn** [comment]
Yeah. But be careful: there aren’t so many really good tutorials.

**Comment by wak3upalex** [comment]
Ok. I got it. I will add a folder or a text file with resources, which i was using for myself
